src/services/persisters/s3Persister.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const fs = require('fs');
+const AWS = require('aws-sdk');
+
+const bucketName = "mirai-app";
+const MongoDbPersister = require('./persisters/mongoDbPersister');
+
+
+class s3Persister {
+    constructor(root) {
```

Comment:
You're passing in root but not doing anything with it at the moment.
Take a look at how other services handle this parameter.
Also, take a look at how other services handle logging. This service should log any errors received from the aws sdk.